tests/test_wsgiref_inputwrapper_with_size.py

```diff
@@ -0,0 +1,32 @@
+import json
+
+import falcon
+from falcon import testing
+
+
+class TypeResource(testing.SimpleTestResource):
+    """ A simple resource to return the posted request body """
+    @falcon.before(testing.capture_responder_args)
+    def on_post(self, req, resp, **kwargs):
+        resp.status = falcon.HTTP_200
+        resp.body = json.dumps({'data': req.stream.read().decode('utf-8')})
+
+
+class TestWsgiRefInputWrapper(testing.TestCase):
+    def setUp(self):
+        super(TestWsgiRefInputWrapper, self).setUp()
+
+        # Set up a route to our TypeResoure
+        self.type_route = '/type'
+        self.api.add_route(self.type_route, TypeResource())
+
+    def test_resources_can_read_request_stream_during_tests(self):
+        """ Make sure we can perform a simple request during testing
+
+        Note: Normally, testing would fail after performing a request because no size is specified
```

Comment:
Style/nit: Although Falcon's pep8 config allows lines over 80 characters long, my rule of thumb is to wrap when you can and only go over for that occasional line of code that would otherwise look really ugly if wrapped.